c2corg_ui/static/js/card.js

```diff
@@ -216,7 +214,22 @@ app.CardController.prototype.getFullRatings = function() {
       }
     }
   }
-  return ratings;
+  var orderedRatings = [ratings['global_rating'], ratings['rock_rating'], ratings['aid_rating'], ratings['ice_rating'],
+    ratings['mixed_rating'], ratings['via_ferrata_rating'], ratings['engagement_rating'], ratings['risk_rating'],
+    ratings['equipment_rating'], ratings['exposition_rock_rating'], ratings['ski_rating'], ratings['labande_rating'],
+    ratings['hiking_rating'], ratings['snowshoe_rating'], ratings['mtb_rating'], doc['hiking_mtb_exposition']];
+
+  // we want to get the name of the prop to show it as a tooltip
+  $.each(orderedRatings, function(i, rating) {
```

Comment:
This code looks a bit complicated and confusing, I think.
And what happens if 2 ratings share the same value (for example 'exposition_rock_rating' and 'hiking_mtb_exposition')?

Isn't it possible to do something like:
```
var ratingOrder = ['global_rating', 'rock_rating', ... ];
ratingOrder.forEach(function(name) {
  if (prop in ratings && ratings[prop]) {
    fullRatings[prop] = ratings[prop];
  }
});
```
or even to use the array ``sort`` function https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Array/sort ?